fades/helpers.py

```diff
@@ -48,3 +50,46 @@ def get_basedir():
         logger.debug("Package xdg not installed; using ~/.fades folder")
         from os.path import expanduser
         return expanduser("~/.fades")
+
+
+def _get_interpreter_info(interpreter=None):
+    """Returns the interpreter's full path using pythonX.Y format and his version"""
+    if interpreter is None:
+        # If interpreter is None by default returns the current interpreter data.
+        major, minor = sys.version_info[:2]
+        executable = sys.executable
+    else:
+        cmd = ("import sys, json; d = dict(path=sys.executable);"
```

Comment:
Build this cmd as a module-level constant, in a multiline string for clarity, like:

SHOW_VERSION_CMD = """
import sys, json
d = dict(path=sys.executable)
d.update(zip('major minor micro releaselevel serial'.split(), sys.version_info))
print(json.dumps(d))
"""